DocGen4/Output.lean

```diff
@@ -61,21 +61,24 @@ def sourceLinker : IO (Name → Option DeclarationRange → String) := do
     | some range => s!"{basic}#L{range.pos.line}-L{range.endPos.line}"
     | none => basic
 
-def htmlOutput (result : AnalyzerResult) (root : String) : IO Unit := do
-  let config := { root := root, result := result, currentName := none, sourceLinker := ←sourceLinker}
-  let basePath := FilePath.mk "." / "build" / "doc"
-  let indexHtml := ReaderT.run index config 
-  let notFoundHtml := ReaderT.run notFound config
+def htmlOutput (result : AnalyzerResult) : IO Unit := do
+  let basePath := FilePath.mk "./build/doc/"
```

Comment:
We should leave this like in the original with the operating systems path separator just in case Windows has some special feelings about this.